packages/sources/cfbenchmarks-test/src/endpoint/rest/crypto.ts

```diff
@@ -0,0 +1,47 @@
+import { BatchWarmingTransport } from '@chainlink/external-adapter-framework/transports'
+import { EmptyObject } from '@chainlink/external-adapter-framework/util'
+import { EndpointTypes } from '../common/crypto'
+
+interface PayloadValue {
+  value: string
+  time: number
+}
+
+interface ProviderResponse {
+  payload: PayloadValue[]
+}
+
+export type RestEndpointTypes = EndpointTypes & {
+  Provider: {
+    RequestBody: EmptyObject
+    ResponseBody: ProviderResponse
+  }
+}
+
+// NOTE: This is using the BatchWarming transport, but the actual API endpoint is not batchable
```

Comment:
I apologize, I missed this earlier. I looked at the cfbenchmarks API and I thought we were using the `latest_values` endpoint that does support batching (although it has a limit on the number of ids we can send). For the priorities relevant to this migration, I think it's fine if we leave this in an incomplete state and throw an error when not enabling websockets, and come back to this EA later on to rework the rest api